recipes/ffmpeg/all/conanfile.py

```diff
@@ -351,41 +352,76 @@ def package(self):
                     for lib in glob.glob("*.a"):
                         tools.rename(lib, lib[3:-2] + ".lib")
 
+    def read_component_version(self, component_name):
```

Comment:
```suggestion
    def _read_component_version(self, component_name):
```

Adding _ would make more clear that it is a private method.